debspawn/build.py

```diff
@@ -147,14 +147,14 @@ def _retrieve_artifacts(osbase, tmp_dir):
     print_section('Retrieving build artifacts')
 
     o_uid, o_gid = get_owner_uid_gid()
-    acount = 0
+    account = 0
```

Comment:
That's not a typo: It's the **a**rtifact **count**, renaming the variable would be wrong in this instance.

Reply:
false positive from codespell...